.travis.yml

```diff
@@ -12,6 +12,7 @@ services:
   - docker
 
 before_script:
+  - npm install sql.js
```

Comment:
why? we don't do same for other drivers. You just need to install sql.js and save into dependencies

Reply:
Yes, you are right